src/Containers/SessionChoice/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Box, OptionalText, Tags } from './styles';
+import Button from 'Components/Button';
+import { Col, Row } from 'antd';
+import Input from 'Components/Input';
```

Comment:
Try to organize your imports
```suggestion
import { Col, Row } from 'antd';

import Button from 'Components/Button';
import Input from 'Components/Input';
import { Box, OptionalText, Tags } from './styles';
```